.gitignore

```diff
@@ -27,3 +27,4 @@ yarn-debug.log*
 yarn-error.log*
 **/.idea/**
 **/.vscode/**
+build_scripts/*.js
```

Comment:
Why is this in ignore?

Reply:
Because the code files are in typescript (.ts) , the build will compile them in .js